tests/integration/test_hyper_transformer.py

```diff
@@ -233,34 +367,6 @@ def test_dtype_category():
     pd.testing.assert_frame_equal(rever, df)
 
 
-def test_empty_transformers():
-    """If transformers is an empty dict, do nothing."""
-    data = get_input_data_without_nan()
-
-    ht = HyperTransformer(transformers={})
-    ht.fit(data)
-
-    transformed = ht.transform(data)
-    reverse = ht.reverse_transform(transformed)
-
-    pd.testing.assert_frame_equal(data, transformed)
-    pd.testing.assert_frame_equal(data, reverse)
-
-
-def test_empty_transformers_nan_data():
-    """If transformers is an empty dict, do nothing."""
-    data = get_input_data_with_nan()
-
-    ht = HyperTransformer(transformers={})
-    ht.fit(data)
-
-    transformed = ht.transform(data)
-    reverse = ht.reverse_transform(transformed)
-
-    pd.testing.assert_frame_equal(data, transformed)
-    pd.testing.assert_frame_equal(data, reverse)
-
-
 def test_subset_of_columns():
```

Comment:
Maybe it would be interesting to add the opposite case: transform and reverse transform data that contains unexpected columns